packages/adblocker-webextension-cosmetics/test/adblocker.test.ts

```diff
@@ -179,6 +179,53 @@ describe('#injectCosmetics', () => {
     expect(dom.window.document.getElementsByTagName('span')).to.have.lengthOf(1);
   });
 
+  it('injects scriptlet in Firefox', async () => {
+    const loader = new ResourceLoader({
+      userAgent: "Firefox"
+    });
+    const dom = new JSDOM(
+      `
+<!DOCTYPE html>
+<head></head>
+<body>
+</body>
+`,
+      {
+        resources: loader,
+        runScripts: 'dangerously',
+      },
+    );
+
+    // JSDOM does not support createObjectURL so we replace blobs with data urls
+    // @ts-ignore
+    dom.window.URL.createObjectURL = async (blob: Blob) => {
```

Comment:
Just as a comment. I assume the data would be eagerly computed in the test to avoid the async callback. Eliminating the need to have the sleep of one second also.

Reply:
we wrap the script with the `autoRemoveScript`, so it wouldn't be one-to-one